matrix/lambdas/api/core.py

```diff
@@ -47,24 +46,26 @@ def post_matrix(body: dict):
                                                 "Visit https://matrix.dev.data.humancellatlas.org for more information."
                                  })
 
+    if not has_url and len(body['bundle_fqids']) > 128000:
```

Comment:
Does this check the length of the array or the size in bytes of `body['bundle_fqids]`?

Reply:
Good call, pretty sure it's the former. I changed it.